packages/angular_devkit/architect/src/create-builder.ts

```diff
@@ -129,6 +129,7 @@ export function createBuilder<
           ) {
             const run = await scheduleByName(builderName, options, {
               scheduler,
+              target: this.target,
```

Comment:
This should not be default behavior.  In the example, the targets happen to align but this cannot be guaranteed.  `target` should be an option on the `ScheduleOptions` interface and default to undefined to maintain existing behavior.

Reply:
I would argue that if you run, e.g. 'ng run project:my-target' then target should always be  `project:my-target`. If you run `ng build` target should always be `default-project:build`. So calling a builder from a custom build should in my opinion pass the target from the custom builder.

It can be made configurable, but in my opinion this should be the default behavior.

Reply:
It's a breaking change to alter the default behavior.
Also a target references a specific configuration within a workspace file.  That configuration may have no relation to a builder that is manually scheduled within another.  Any number or types of builders may also be scheduled within one builder and a builder may act differently depending upon whether it is executed from a workspace target or not.

The exception that is attempting to be bypassed is also an implementation choice of the particular builder being scheduled (`@angular-devkit/build-angular:browser`) and not an issue with the system in a general case.

Reply:
Ok, I put it to `ScheduleOptions` to make it configurable.